Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -0,0 +1,272 @@
+    channels:
+    - Common
+    - Command
+    - Medical
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cmo_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyMedicalScience
+  name: med-sci encryption key
+  description: An encryption key used by someone who hasn't decided which side to take.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Medical
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: medsci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyScience
+  name: science encryption key
+  description: An encryption key used by scientists. Maybe it is plasmaproof?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: sci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyRobo
+  name: robotech encryption key
+  description: An encryption key used by robototech engineers. Maybe it has a LAH-6000 on it?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: rob_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyRD
+  name: research director's encryption key
+  description: An encryption key used by Fordon Griman. Seems to have signs of something red, probably tomato juice?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: rd_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeySecurity
+  name: security encryption key
+  description: An encryption key used by security.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Security
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: sec_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyHOS
+  name: head of security encryption key
+  description: An encryption key used by the boss of security.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Security
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: hos_cypherkey
+
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyService
+  name: service encryption key
+  description: An encryption key used by the service staff, tasked with keeping the station full, happy and clean.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Service
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: srv_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeySyndie
+  name: blood-red encryption key
+  description: An encryption key used by... wait... Who is owner of this chip?
```

Comment:
```suggestion
  name: syndicate encryption key
  description: An encryption key used by Syndicate personnel for securely contacting fellow agents and listening into departmental channels.
```

Reply:
You can't get know from looking at key which organisation it's belongs, only if it have large logo at the backside "SYNDICATE PROPERTY, RETURN TO: Solar System, Luna, Seleste, Syndieratt yard, House 11"